.github/workflows/auto-release-reusable.yml

```diff
@@ -0,0 +1,34 @@
+name: "auto-release-reusable"
+on:
+  workflow_call:
+    inputs:
+      actions-files-checkout-path:
+        description: 'The path to checkout the action scripts to at runtime'
+        type: string
+        default: github-action-auto-release
+        required: false
+      publish:
+        description: "Whether to publish a new release immediately"
+        type: boolean
+        required: false
+        default: true
+      prerelease:
+        description: "Boolean indicating whether this release should be a prerelease"
+        type: boolean
+        required: false
+        default: false
+    secrets:
+      token:
+        description: "Standard GitHub token (e.g., secrets.GITHUB_TOKEN)"
```

Comment:
Consider updating this to be more descriptive...try to answer the question when do I have to provide this token and what is it used for? Isn't this always required to be able to create a release?